setup.py

```diff
@@ -66,7 +68,12 @@
     url="http://www.reana.io/",
     packages=packages,
     zip_safe=False,
-    entry_points={"console_scripts": ["reana-dev = reana.reana_dev.cli:reana_dev",],},
+    entry_points={
+        "console_scripts": [
+            "reana-dev = reana.reana_dev.cli:reana_dev",
+            "reana-bench = reana.reana_bench.cli:reana_bench",
```

Comment:
Question: What about renaming to `reana-benchmark`?  It's longer, but more precise and unambiguous.

Reply:
Changed entrypoints + updated `reana-bench` to `reana-benchmark` accross files.